README.md

```diff
@@ -32,7 +106,10 @@ If you want the console to run in `https` mode, run:
 This command will run the console in `https` mode on port 9442.
 The console url is <https://localhost:9442>
 
-Note: Running console in `https` mode requires the plugin running in `https` mode too.
+> [!IMPORTANT]
+> Running console in `https` mode requires the plugin running in `https` mode too.
```

Comment:
@lavocatt , currently as our plugin by default run in the https mode we don't need to add this info right?

Reply:
That's a good question. But it might be out of the scope of the PR since this is related to the OpenShift setup. Maybe a followup would make sense?

Reply:
sure we can. can i update that?

Reply:
Yes ofc :slightly_smiling_face: